### PR TITLE
food icon changes

### PR DESCRIPTION
**Pull request by lroche823**
for issue #502 
besides what is already suggested in the issue, i added an extra icon for baby formula and changed the protein icon to something that fit the aesthetic of the other icons a little better.

### CODE REVIEW

File: src/Components/WICScreen.jsx [hunk 1]
```diff
@@ -260,35 +265,35 @@ export const wicFeeding = () => {
             subtext={
               '24-32 ounces\nOr based on individual nutritional assessment'
             }
-            icon={breastfeeding}
+            icon={formula}
```
Comment (dannydaking23) [inline src/Components/WICScreen.jsx:268]: New icon for baby formula is ideal for the app and not only shows an example of animated baby formula, but also fits the theme of the app very well.

File: src/Components/WICScreen.jsx [hunk 2]
```diff
@@ -260,35 +265,35 @@ export const wicFeeding = () => {
...
           />
           <BetterMenu
             style={appStyles.ImageOnRightSelectionButton}
             text="Grain Products"
             subtext={
               '1-2 ounces\nIron-fortified infant cereals, bread, small pieces of cracker'
             }
-            icon={breastfeeding}
+            icon={grains}
```
Comment (dannydaking23) [inline src/Components/WICScreen.jsx:276]: New icon for grains displays an animated bag of wheat which is a simple and effective display for the app.

File: src/Components/WICScreen.jsx [hunk 3]
```diff
@@ -260,35 +265,35 @@ export const wicFeeding = () => {
...
           />
           <BetterMenu
             style={appStyles.ImageOnRightSelectionButton}
             text="Vegetables"
             subtext={'2-4 ounces\nCooked, plain strained/pureed/mashed'}
-            icon={breastfeeding}
+            icon={vegetables}
```
Comment (dannydaking23) [inline src/Components/WICScreen.jsx:282]: New icon for the vegetables is also a simple and animated portrayal for the vegetables food group and fits the applications theme.

File: src/Components/WICScreen.jsx [hunk 4]
```diff
@@ -260,35 +265,35 @@ export const wicFeeding = () => {
...
           />
           <BetterMenu
             style={appStyles.ImageOnRightSelectionButton}
             text="Fruits"
             subtext={'2-4 ounces\nPlain strained/pureed/mashed'}
-            icon={breastfeeding}
+            icon={fruits}
```
Comment (dannydaking23) [inline src/Components/WICScreen.jsx:288]: Fruits icon displays all of the common fruits typically mentioned when discussing this food group and blends sell with the app and other food groups.

File: src/Components/WICScreen.jsx [hunk 5]
```diff
@@ -260,35 +265,35 @@ export const wicFeeding = () => {
...
           />
           <BetterMenu
             style={appStyles.ImageOnRightSelectionButton}
             text="Protein-rich Foods"
             subtext={
               '1-2 ounces\nPlain strained/pureed/mashed meat, poultry, fish, eggs, cheese, yogurt, or mashed legumes'
             }
-            icon={breastfeeding}
+            icon={proteins}
```
Comment (dannydaking23) [inline src/Components/WICScreen.jsx:296]: New icon for the proteins food group neatly shows different forms of protein in a simple animated manner which fits the app well.

**Review by dannydaking23 — APPROVED** [review]
All of the changes to the icons are ideal for the app and achieve the goal of portraying the different food groups as images.

**Review by Elijahx — APPROVED** [review]
All the icon changes look good to go. Well executed.

**Pull request merged**